base/commands/shell.go

```diff
@@ -103,6 +105,33 @@ func (cm *ShellCommand) ExecInteractive(ctx context.Context, ec plug.ExecContext
 	return sh.Start(ctx)
 }
 
+func prompt(cfgPath string) string {
+	if cfgPath == "" {
+		return "CLC> "
```

Comment:
`cfgPath` should never be a blank string, but just in case it is...
```suggestion
		return "> "
```